cccatalog-api/cccatalog/api/utils/ccrel.py

```diff
@@ -0,0 +1,74 @@
+from libxmp.consts import XMP_NS_CC, XMP_NS_XMP_Rights, XMP_NS_XMP
+import libxmp
+import io
+import os
+import uuid
+
+"""
+Tools for embedding Creative Commons Rights Expression Language (ccREL) data
```

Comment:
This would be nice to release as a standalone library but that's out of scope for this PR.

Reply:
I think you're right, especially since the raw XMP tools are a little bit difficult to use and sparsely documented. Once this module passes a certain size/complexity threshold I'll revisit separating it into a library.

Reply:
Could you create an issue to track that so we don't forget about it?

Reply:
Yep - https://github.com/creativecommons/cccatalog-api/issues/248